src/scss/FeedCard.scss

```diff
@@ -1,6 +1,15 @@
-@import "../theme/index.scss";
-
-.leftTop {
-  @include flex(column, space-between, align-items);
-  background-color: #ff0000;
-}
+<<<<<<< HEAD
+@import "../theme/index.scss";
+
+.leftTop {
+  @include flex(column, space-between, align-items);
+  background-color: #ff0000;
+}
+=======
```

Comment:
This looks like you are in the middle of a merge. If that's not the case, please remove the `<<<<`, `====`, and `>>>>` lines.